src/js/modules/__mocks__/load-counter-home.js

```diff
@@ -0,0 +1,3 @@
+const loadBreeds = () => {};
+
+export default loadBreeds;
```

Comment:
Ideally, we should try to avoid mocking this out. If the problem is about loading resources, we could try to figure that out later.

Reply:
Yes, I got some issues here maybe we could do it together, thank you!